packages/tests/playwright.config.ts

```diff
@@ -44,15 +47,15 @@ export default defineConfig({
       use: { ...devices["Desktop Chrome"] },
     },
 
-    {
+    /* {
```

Comment:
Double checking: Do we really want those disabled?

Reply:
I disabled them at least until code review is done in order to iterate quickly. We still have to discuss the entire testing matrix (browser, build tool, UI rendering framework) - these multiply the number of runs so I wanted to check with the team what they think about the trade offs.